src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -51,141 +52,1161 @@
     static boolean isDenaliOrLater = false;
 
     @Test
-    public void testPointWkb() throws SQLException {
+    public void testPoint() throws SQLException {
         String geoWKT = "POINT(3 40 5 6)";
-        byte[] geomWKB = hexStringToByteArray(
+        byte[] geomCLR = hexStringToByteArray(
                 "00000000010F0000000000000840000000000000444000000000000014400000000000001840");
-        byte[] geogWKB = hexStringToByteArray(
+        byte[] geogCLR = hexStringToByteArray(
                 "E6100000010F0000000000004440000000000000084000000000000014400000000000001840");
-        Geometry geomWKT = Geometry.deserialize(geomWKB);
-        Geography geogWKT = Geography.deserialize(geogWKB);
+        byte[] geoWKB = hexStringToByteArray("010100000000000000000008400000000000004440");
+        Geometry geomWKT = Geometry.deserialize(geomCLR);
+        Geography geogWKT = Geography.deserialize(geogCLR);
+        assertEquals(geomWKT.asTextZM(), geoWKT);
+        assertEquals(geogWKT.asTextZM(), geoWKT);
+        assertTrue(Arrays.equals(geomWKT.STAsBinary(), geoWKB));
+        assertTrue(Arrays.equals(geogWKT.STAsBinary(), geoWKB));
+    }
+
+    @Test
+    public void testPoint2() throws SQLException {
```

Comment:
all these testsBlah and testBlah2 tests are identical other than test data, can we refactor them to call a common test method instead?